.kokoro/tests/run_tests.sh

```diff
@@ -76,6 +76,16 @@ for file in **/requirements.txt; do
         fi
     fi
 
+    # Skip unsupported Python versions for Cloud Functions
+    # (Some GCF samples' dependencies don't support them)
+    if [[ "$file" == "functions/"* ]]; then
+      PYTHON_VERSION="$(python --version 2>&1)"
```

Comment:
FYI, this code is completely wrong.

This code checks the default Python version, rather than Python version in the nox session.

This resulted in disabling all the tests under functions directory. #3767 removed this code block then our tests started to fail.